mm2src/coins/nft/storage/wasm/wasm_storage.rs

```diff
@@ -227,9 +645,42 @@ impl TableSignature for NftTxHistoryTable {
     fn on_upgrade_needed(upgrader: &DbUpgrader, old_version: u32, new_version: u32) -> OnUpgradeResult<()> {
         if let (0, 1) = (old_version, new_version) {
             let table = upgrader.create_table(Self::table_name())?;
+            table.create_multi_index(
+                Self::CHAIN_TOKEN_ADD_TOKEN_ID_INDEX,
+                &["chain", "token_address", "token_id"],
+                false,
+            )?;
             table.create_multi_index(Self::CHAIN_TX_HASH_INDEX, &["chain", "transaction_hash"], true)?;
+            table.create_multi_index(Self::CHAIN_BLOCK_NUMBER_INDEX, &["chain", "block_number"], false)?;
```

Comment:
same as above

Reply:
It is possible that several tx will be included in one block.

Reply:
ah okay thanks for the info